apps/ios/Shared/Views/Call/ActiveCallView.swift

```diff
@@ -130,14 +150,16 @@ struct ActiveCallView: View {
                     call.localCamera = camera
                     Task {
                         // This disables microphone if it was disabled before flipping the camera
-                        await webView.setMicrophoneCaptureState(call.audioEnabled ? .active : .muted)
+                        client.setAudioEnabled(call.audioEnabled)
```

Comment:
on lines 148 (call.callState = .negotiated) and 150 (call.localCamera = camera):
/Users/evgeny/code/simplex-chat/simplex-chat/apps/ios/Shared/Views/Call/ActiveCallView.swift:148: warning run: Publishing changes from background threads is not allowed; make sure to publish values from the main thread (via operators like receive(on:)) on model updates.